### PR TITLE
request slots for art slab

### PR DESCRIPTION
**Pull request by nmushegian**
Is this approach ok?
>inb4 failing CI below

### CODE REVIEW

**Comment by nmushegian** [comment]
context:
https://www.reddit.com/r/MakerDAO/comments/b5s0al/mkr_roll_call_vote_in_1_week_serious_vote_fun/
https://www.reddit.com/r/MakerDAO/comments/b66y32/example_result_of_slab_etchings/

File: src/vat.sol [hunk 1]
```diff
@@ -50,6 +50,9 @@ contract Vat {
     mapping (address => uint256)                   public dai;  // [rad]
     mapping (address => uint256)                   public sin;  // [rad]
 
+    bytes32 public slab = 0x0;  // Maker Art IPFS Hash
+    bytes32 public rest = 0x0;  // First 32+8 bytes
```
Comment (godsflaw) [inline src/vat.sol]: I see you removed the ability to set these from the constructor, are we just waiting for another commit where you will either hardcode values or add a setter?

Comment (nmushegian) [reply src/vat.sol]: Will make another commit when we have the values

**Comment by BIGCRYPTO001** [comment]
Please Sucure This Address
0xb7bBF6D2f5372022cc07B6677A9eB49ce78dA8c3

**Comment by livnev** [comment]
@nmushegian would be more convenient if you added these as the last storage variables declared, since then the storage layout won't need to be remapped for FV.

**Pull request closed without merging**